### ISSUE TITLE
Loved music not reflecting on app as it is on web

### ISSUE THREAD

**Issue by Olamatto** [DDihanov/musiQ#19]
#### Expected behavior
All details on an account on the website, must be the same as what will be shown in the application. Loved tracks on the website, will also reflect ln the application. 

#### Actual behavior
After adding a track to loved track on the website, it does not reflect in the application. 

#### How to reproduce
- Download and install the application
- Switch on your data connection
- Launch the application
- Log in if you already have an account or click on the sign up button if you are yet to get an account 

<hr>

- Visit https://www.last.fm/music, love some tracks
- Click on your profile icon,  select loved tracks,  to view loved tracks on the website. 
- Go back to the app
- Log in
- Click on the menu button, by the top right corner 
- Click on the profile section 
- Click on the Loved track 

And note the bug there. 

* App version: musiQ 1.845
* Operating system: Android v7.0

#### Recording Of The Bug
https://youtu.be/VssF3XO3E_U

**Comment by DDihanov** [comment]
Hello. Thanks for the report. I tested this on several devices and it seemed to work on all of them. Can you please try again? It might be on last.fm's side and not on mine. Maybe there was a problem with the website at the time.

**Comment by Olamatto** [comment]
Did you try it on Android v7.0?

**Comment by DDihanov** [comment]
Yes I did. Still works. Can you see if the issue still persists?

**Comment by Olamatto** [comment]
I updated the app, but it not stable,  it sometimes show the loved tracks, even my profile only shows the link nothing else.
